src/promptflow/promptflow/_sdk/data/docker/start.sh.jinja2

```diff
@@ -1,16 +1,16 @@
-# Startup script for the flow container
+#!/bin/bash
 
-CONDA_ENV_PATH="$(conda info --base)/envs/{{env.conda_env_name}}"
-export PATH="$CONDA_ENV_PATH/bin:$PATH"
+# stop services created by runsv and propagate SIGINT, SIGTERM to child jobs
+sv_stop() {
```

Comment:
could you please run local test offline to include a new version of generated snapshot?

Reply:
updated